### PR TITLE
Pioneer 16 Leg 2 PM deploy/recover updates

### PR DESCRIPTION
**Pull request by snwhite**
Deployment and recovery times/locations come from the Pioneer 16 Leg 2 Quick Look Report (Rebecca posted a draft to the Pioneer 16 Slack channel).
Depths for components on Profiler Moorings are captured in the CGSN_Mooring_Component_Depths.xlsx in Vault (Project_Files > Documentation > System > System Notebook > Asset Management).

CNPM, PMCO, PMUO and OSPM have been posted first. The rest will be posted later.

### CODE REVIEW

**Comment by snwhite** [comment]
PMCI, PMUI and ISPM updates have been added.

File: deployment/CP01CNPM_Deploy.csv [hunk 1]
```diff
@@ -26,12 +26,12 @@ AR39,,AR44,,CP01CNPM-WFP01-02-DOFSTK000,3,1,2019-10-07T13:44:00,2020-06-14T12:23
...
+AR48,,AR52,,CP01CNPM-WFP01-00-WFPENG000,4,1,2020-10-29T20:10:00,2021-04-10T20:29:00,CGMCP-01CNPM-00004,CGVEH-CWFPCP-30002,CGCON-CWFPCP-30002,40.134012,-70.770805,,112,133,,,
+AR48,,AR52,,CP01CNPM-WFP01-01-VEL3DK000,4,1,2020-10-29T20:10:00,2021-04-10T20:29:00,CGMCP-01CNPM-00004,CGVEH-CWFPCP-30002,CGINS-VEL3DK-00470,40.134012,-70.770805,,112,133,,,
+AR48,,AR52,,CP01CNPM-WFP01-02-DOFSTK000,4,1,2020-10-29T20:10:00,2021-04-10T20:29:00,CGMCP-01CNPM-00004,CGVEH-CWFPCP-30002,CGINS-DOFSTK-03523,40.134012,-70.770805,,112,133,,,
+AR48,,AR52,,CP01CNPM-WFP01-03-CTDPFK000,4,1,2020-10-29T20:10:00,2021-04-10T20:29:00,CGMCP-01CNPM-00004,CGVEH-CWFPCP-30002,CGINS-CTDPFK-00185,40.134012,-70.770805,,112,133,,,
+AR48,,AR52,,CP01CNPM-WFP01-04-FLORTK000,4,1,2020-10-29T20:10:00,2021-04-10T20:29:00,CGMCP-01CNPM-00004,CGVEH-CWFPCP-30002,CGINS-FLORTK-01538,40.134012,-70.770805,,112,133,,,
+AR48,,AR52,,CP01CNPM-WFP01-05-PARADK000,4,1,2020-10-29T20:10:00,2021-04-10T20:29:00,CGMCP-01CNPM-00004,CGVEH-CWFPCP-30002,CGINS-PARADK-20609,40.134012,-70.770805,,112,133,,,
```
Comment (cdobs) [inline deployment/CP01CNPM_Deploy.csv:37]: Looks good

File: deployment/CP02PMUI_Deploy.csv [hunk 2]
```diff
@@ -134,12 +134,12 @@ AR44,,,,CP02PMUI-WFP01-02-DOFSTK000,15,1,2020-06-13T13:47:00,2020-12-08T06:30:00
...
+AR52,,,,CP02PMUI-SBS01-00-STCENG000,16,1,2021-04-12T13:49:00,,CGMCP-02PMUI-00016,,CGCON-BSTC01-00042,40.365183,-70.7697,,0,95,,,
+AR52,,,,CP02PMUI-SBS01-01-MOPAK0000,16,1,2021-04-12T13:49:00,,CGMCP-02PMUI-00016,,CGINS-MOPAK0-43252,40.365183,-70.7697,,0,95,,,
+AR52,,,,CP02PMUI-WFP01-00-WFPENG000,16,1,2021-04-12T13:49:00,,CGMCP-02PMUI-00016,CGVEH-CWFPCP-71401,CGCON-CWFPCP-71401,40.365183,-70.7697,,71,95,,,
+AR52,,,,CP02PMUI-WFP01-01-VEL3DK000,16,1,2021-04-12T13:49:00,,CGMCP-02PMUI-00016,CGVEH-CWFPCP-71401,CGINS-VEL3DK-00012,40.365183,-70.7697,,71,95,,,
+AR52,,,,CP02PMUI-WFP01-02-DOFSTK000,16,1,2021-04-12T13:49:00,,CGMCP-02PMUI-00016,CGVEH-CWFPCP-71401,CGINS-DOFSTK-00209,40.365183,-70.7697,,71,95,,,
+AR52,,,,CP02PMUI-WFP01-03-CTDPFK000,16,1,2021-04-12T13:49:00,,CGMCP-02PMUI-00016,CGVEH-CWFPCP-71401,CGINS-CTDPFK-00093,40.365183,-70.7697,,71,95,,,
+AR52,,,,CP02PMUI-WFP01-04-FLORTK000,16,1,2021-04-12T13:49:00,,CGMCP-02PMUI-00016,CGVEH-CWFPCP-71401,CGINS-FLORTK-00830,40.365183,-70.7697,,71,95,,,
+AR52,,,,CP02PMUI-WFP01-05-PARADK000,16,1,2021-04-12T13:49:00,,CGMCP-02PMUI-00016,CGVEH-CWFPCP-71401,CGINS-PARADK-20525,40.365183,-70.7697,,71,95,,,
```
Comment (cdobs) [inline deployment/CP02PMUI_Deploy.csv:145]: Looks good

File: deployment/CP02PMCO_Deploy.csv [hunk 3]
```diff
@@ -116,21 +116,21 @@ AR39,,AR44,,CP02PMCO-WFP01-02-DOFSTK000,13,1,2019-10-08T02:02:00,2020-06-10T16:0
...
+AR52,,,,CP02PMCO-SBS01-00-STCENG000,15,1,2021-04-11T13:44:00,,CGMCP-02PMCO-00015,,CGCON-BSTC01-00016,40.09645,-70.879083,,0,148,,,
+AR52,,,,CP02PMCO-SBS01-01-MOPAK0000,15,1,2021-04-11T13:44:00,,CGMCP-02PMCO-00015,,CGINS-MOPAK0-43263,40.09645,-70.879083,,0,148,,,
+AR52,,,,CP02PMCO-WFP01-00-WFPENG000,15,1,2021-04-11T13:44:00,,CGMCP-02PMCO-00015,CGVEH-CWFPCP-99102,CGCON-CWFPCP-99102,40.09645,-70.879083,,127,148,,,
+AR52,,,,CP02PMCO-WFP01-01-VEL3DK000,15,1,2021-04-11T13:44:00,,CGMCP-02PMCO-00015,CGVEH-CWFPCP-99102,CGINS-VEL3DK-00013,40.09645,-70.879083,,127,148,,,
+AR52,,,,CP02PMCO-WFP01-02-DOFSTK000,15,1,2021-04-11T13:44:00,,CGMCP-02PMCO-00015,CGVEH-CWFPCP-99102,CGINS-DOFSTK-02499,40.09645,-70.879083,,127,148,,,
+AR52,,,,CP02PMCO-WFP01-03-CTDPFK000,15,1,2021-04-11T13:44:00,,CGMCP-02PMCO-00015,CGVEH-CWFPCP-99102,CGINS-CTDPFK-00113,40.09645,-70.879083,,127,148,,,
+AR52,,,,CP02PMCO-WFP01-04-FLORTK000,15,1,2021-04-11T13:44:00,,CGMCP-02PMCO-00015,CGVEH-CWFPCP-99102,CGINS-FLORTK-01031,40.09645,-70.879083,,127,148,,,
+AR52,,,,CP02PMCO-WFP01-05-PARADK000,15,1,2021-04-11T13:44:00,,CGMCP-02PMCO-00015,CGVEH-CWFPCP-99102,CGINS-PARADK-20447,40.09645,-70.879083,,127,148,,,
```
Comment (cdobs) [inline deployment/CP02PMCO_Deploy.csv:136]: Looks good

File: deployment/CP02PMCI_Deploy.csv [hunk 4]
```diff
@@ -116,21 +116,21 @@ AR39,,AR44,,CP02PMCI-WFP01-02-DOFSTK000,13,1,2019-10-07T18:23:00,2020-06-12T16:0
...
+AR52,,,,CP02PMCI-SBS01-00-STCENG000,15,1,2021-04-13T13:35:00,,CGMCP-02PMCI-00015,,CGCON-BSTC01-00017,40.226983,-70.878183,,0,128,,,
+AR52,,,,CP02PMCI-SBS01-01-MOPAK0000,15,1,2021-04-13T13:35:00,,CGMCP-02PMCI-00015,,CGINS-MOPAK0-58478,40.226983,-70.878183,,0,128,,,
+AR52,,,,CP02PMCI-WFP01-00-WFPENG000,15,1,2021-04-13T13:35:00,,CGMCP-02PMCI-00015,CGVEH-CWFPCP-99101,CGCON-CWFPCP-99101,40.226983,-70.878183,,105,128,,,
+AR52,,,,CP02PMCI-WFP01-01-VEL3DK000,15,1,2021-04-13T13:35:00,,CGMCP-02PMCI-00015,CGVEH-CWFPCP-99101,CGINS-VEL3DK-00015,40.226983,-70.878183,,105,128,,,
+AR52,,,,CP02PMCI-WFP01-02-DOFSTK000,15,1,2021-04-13T13:35:00,,CGMCP-02PMCI-00015,CGVEH-CWFPCP-99101,CGINS-DOFSTK-02978,40.226983,-70.878183,,105,128,,,
+AR52,,,,CP02PMCI-WFP01-03-CTDPFK000,15,1,2021-04-13T13:35:00,,CGMCP-02PMCI-00015,CGVEH-CWFPCP-99101,CGINS-CTDPFK-00111,40.226983,-70.878183,,105,128,,,
+AR52,,,,CP02PMCI-WFP01-04-FLORTK000,15,1,2021-04-13T13:35:00,,CGMCP-02PMCI-00015,CGVEH-CWFPCP-99101,CGINS-FLORTK-01029,40.226983,-70.878183,,105,128,,,
+AR52,,,,CP02PMCI-WFP01-05-PARADK000,15,1,2021-04-13T13:35:00,,CGMCP-02PMCI-00015,CGVEH-CWFPCP-99101,CGINS-PARADK-20541,40.226983,-70.878183,,105,128,,,
```
Comment (cdobs) [inline deployment/CP02PMCI_Deploy.csv:136]: Looks good

File: deployment/CP02PMUO_Deploy.csv [hunk 5]
```diff
@@ -134,21 +134,21 @@ AR44,,AR48,,CP02PMUO-WFP01-02-DOFSTK000,15,1,2020-06-09T13:57:00,2020-11-09T13:1
...
+AR52,,,,CP02PMUO-SBS01-00-STCENG000,17,1,2021-04-10T13:41:00,,CGMCP-02PMUO-00017,,CGCON-BSTC01-00043,39.943183,-70.77105,,0,445,,,
+AR52,,,,CP02PMUO-SBS01-01-MOPAK0000,17,1,2021-04-10T13:41:00,,CGMCP-02PMUO-00017,,CGINS-MOPAK0-70027,39.943183,-70.77105,,0,445,,,
+AR52,,,,CP02PMUO-WFP01-00-WFPENG000,17,1,2021-04-10T13:41:00,,CGMCP-02PMUO-00017,CGVEH-CWFPCP-10304,CGCON-CWFPCP-10304,39.943183,-70.77105,,421,445,,,
+AR52,,,,CP02PMUO-WFP01-01-VEL3DK000,17,1,2021-04-10T13:41:00,,CGMCP-02PMUO-00017,CGVEH-CWFPCP-10304,CGINS-VEL3DK-00033,39.943183,-70.77105,,421,445,,,
+AR52,,,,CP02PMUO-WFP01-02-DOFSTK000,17,1,2021-04-10T13:41:00,,CGMCP-02PMUO-00017,CGVEH-CWFPCP-10304,CGINS-DOFSTK-02726,39.943183,-70.77105,,421,445,,,
+AR52,,,,CP02PMUO-WFP01-03-CTDPFK000,17,1,2021-04-10T13:41:00,,CGMCP-02PMUO-00017,CGVEH-CWFPCP-10304,CGINS-CTDPFK-00127,39.943183,-70.77105,,421,445,,,
+AR52,,,,CP02PMUO-WFP01-04-FLORTK000,17,1,2021-04-10T13:41:00,,CGMCP-02PMUO-00017,CGVEH-CWFPCP-10304,CGINS-FLORTK-01100,39.943183,-70.77105,,421,445,,,
+AR52,,,,CP02PMUO-WFP01-05-PARADK000,17,1,2021-04-10T13:41:00,,CGMCP-02PMUO-00017,CGVEH-CWFPCP-10304,CGINS-PARADK-20582,39.943183,-70.77105,,421,445,,,
```
Comment (cdobs) [inline deployment/CP02PMUO_Deploy.csv:154]: Looks good

File: deployment/CP03ISPM_Deploy.csv [hunk 6]
```diff
@@ -26,12 +26,12 @@ AR39,,AR44,,CP03ISPM-WFP01-02-DOFSTK000,3,1,2019-10-06T13:41:00,2020-06-13T19:24
...
+AR48,,AR52,,CP03ISPM-SBS01-01-MOPAK0000,4,1,2020-10-31T22:02:00,2021-04-12T16:31:00,CGMCP-03ISPM-00004,,CGINS-MOPAK0-58481,40.36202,-70.878487,,0,92,,,
+AR48,,AR52,,CP03ISPM-WFP01-00-WFPENG000,4,1,2020-10-31T22:02:00,2021-04-12T16:31:00,CGMCP-03ISPM-00004,CGVEH-CWFPCP-30001,CGCON-CWFPCP-30001,40.36202,-70.878487,,68,92,,,
+AR48,,AR52,,CP03ISPM-WFP01-01-VEL3DK000,4,1,2020-10-31T22:02:00,2021-04-12T16:31:00,CGMCP-03ISPM-00004,CGVEH-CWFPCP-30001,CGINS-VEL3DK-00468,40.36202,-70.878487,,68,92,,,
+AR48,,AR52,,CP03ISPM-WFP01-02-DOFSTK000,4,1,2020-10-31T22:02:00,2021-04-12T16:31:00,CGMCP-03ISPM-00004,CGVEH-CWFPCP-30001,CGINS-DOFSTK-03506,40.36202,-70.878487,,68,92,,,
+AR48,,AR52,,CP03ISPM-WFP01-03-CTDPFK000,4,1,2020-10-31T22:02:00,2021-04-12T16:31:00,CGMCP-03ISPM-00004,CGVEH-CWFPCP-30001,CGINS-CTDPFK-00184,40.36202,-70.878487,,68,92,,,
+AR48,,AR52,,CP03ISPM-WFP01-04-FLORTK000,4,1,2020-10-31T22:02:00,2021-04-12T16:31:00,CGMCP-03ISPM-00004,CGVEH-CWFPCP-30001,CGINS-FLORTK-01537,40.36202,-70.878487,,68,92,,,
+AR48,,AR52,,CP03ISPM-WFP01-05-PARADK000,4,1,2020-10-31T22:02:00,2021-04-12T16:31:00,CGMCP-03ISPM-00004,CGVEH-CWFPCP-30001,CGINS-PARADK-20608,40.36202,-70.878487,,68,92,,,
```
Comment (cdobs) [inline deployment/CP03ISPM_Deploy.csv:37]: Looks good

File: deployment/CP04OSPM_Deploy.csv [hunk 7]
```diff
@@ -100,21 +100,21 @@ AR39,,AR44,,CP04OSPM-WFP01-02-DOFSTK000,12,1,2019-10-13T13:42:00,2020-06-08T16:1
...
+AR52,,,,CP04OSPM-SBS01-00-STCENG000,14,1,2021-04-09T14:05:00,,CGMCP-04OSPM-00014,,CGCON-BSTC01-00007,39.935933,-70.88015,,0,456,,,
+AR52,,,,CP04OSPM-SBS11-02-MOPAK0000,14,1,2021-04-09T14:05:00,,CGMCP-04OSPM-00014,,CGINS-MOPAK0-11347,39.935933,-70.88015,,0,456,,,
+AR52,,,,CP04OSPM-WFP01-00-WFPENG000,14,1,2021-04-09T14:05:00,,CGMCP-04OSPM-00014,CGVEH-CWFPCP-99105,CGCON-CWFPCP-99105,39.935933,-70.88015,,410,456,,,
+AR52,,,,CP04OSPM-WFP01-01-VEL3DK000,14,1,2021-04-09T14:05:00,,CGMCP-04OSPM-00014,CGVEH-CWFPCP-99105,CGINS-VEL3DK-00212,39.935933,-70.88015,,410,456,,,
+AR52,,,,CP04OSPM-WFP01-02-DOFSTK000,14,1,2021-04-09T14:05:00,,CGMCP-04OSPM-00014,CGVEH-CWFPCP-99105,CGINS-DOFSTK-03370,39.935933,-70.88015,,410,456,,,
+AR52,,,,CP04OSPM-WFP01-03-CTDPFK000,14,1,2021-04-09T14:05:00,,CGMCP-04OSPM-00014,CGVEH-CWFPCP-99105,CGINS-CTDPFK-00169,39.935933,-70.88015,,410,456,,,
+AR52,,,,CP04OSPM-WFP01-04-FLORTK000,14,1,2021-04-09T14:05:00,,CGMCP-04OSPM-00014,CGVEH-CWFPCP-99105,CGINS-FLORTK-01466,39.935933,-70.88015,,410,456,,,
+AR52,,,,CP04OSPM-WFP01-05-PARADK000,14,1,2021-04-09T14:05:00,,CGMCP-04OSPM-00014,CGVEH-CWFPCP-99105,CGINS-PARADK-20542,39.935933,-70.88015,,410,456,,,
```
Comment (cdobs) [inline deployment/CP04OSPM_Deploy.csv:120]: Looks good

**Review by cdobs — APPROVED** [review]
Everything looks good to me

File: deployment/CP01CNPM_Deploy.csv (hunk 1 above)
Comment (reedan88) [reply deployment/CP01CNPM_Deploy.csv:37]: Looks good.

File: deployment/CP02PMCI_Deploy.csv (hunk 4 above)
Comment (reedan88) [reply deployment/CP02PMCI_Deploy.csv:136]: Looks good.

File: deployment/CP02PMCO_Deploy.csv (hunk 3 above)
Comment (reedan88) [reply deployment/CP02PMCO_Deploy.csv:136]: Looks good.

File: deployment/CP02PMUI_Deploy.csv (hunk 2 above)
Comment (reedan88) [reply deployment/CP02PMUI_Deploy.csv:145]: Looks good.

File: deployment/CP02PMUO_Deploy.csv (hunk 5 above)
Comment (reedan88) [reply deployment/CP02PMUO_Deploy.csv:154]: Looks good.

File: deployment/CP03ISPM_Deploy.csv (hunk 6 above)
Comment (reedan88) [reply deployment/CP03ISPM_Deploy.csv:37]: Looks good.

File: deployment/CP04OSPM_Deploy.csv (hunk 7 above)
Comment (reedan88) [reply deployment/CP04OSPM_Deploy.csv:120]: Looks good.

**Review by reedan88 — APPROVED** [review]
Everything looks good.

**Pull request merged**